.github/workflows/release.yml

```
@@ -119,21 +67,19 @@ jobs:
+          # TODO: Remove once Flutter's libdartjni.so no longer embeds a non-reproducible build ID
+          sed -i -e 's/-Wl,/-Wl,--build-id=none,/' ${PUB_CACHE:-$HOME/.pub-cache}/hosted/*/jni-*/src/CMakeLists.txt
+          sed -i 's/signingConfig = signingConfigs.getByName("release")//g' android/app/build.gradle.kts
+          flutter build apk --flavor normal && flutter build apk --split-per-abi --flavor normal
+          for file in build/app/outputs/flutter-apk/app-*normal*.apk*; do mv "$file" "${file//-normal/}"; done
+          flutter build apk --flavor fdroid -t lib/main_fdroid.dart && flutter build apk --split-per-abi --flavor fdroid -t lib/main_fdroid.dart
+          rm ./build/app/outputs/flutter-apk/*.sha1
+          cp ./sign.sh ./build/app/outputs/flutter-apk/
+          ls -l ./build/app/outputs/flutter-apk/
+
+      - name: Save Unsigned APKs as Action Artifacts
+        uses: actions/upload-artifact@v4
```

actionlint (1.7.12)

[error] missing input "path" which is required by action "actions/upload-artifact@v4". all required inputs are "path"

(action)